uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/controllers/sub-controllers/leave-request-ctrl_test.js

```diff
@@ -1475,17 +1475,22 @@
        */
       function setTestDates (from, to) {
         if (from) {
-          $ctrl.uiOptions.fromDate = new Date(from);
+          $ctrl.uiOptions.fromDate = getUTCDate(from);
           $ctrl.updateAbsencePeriodDatesTypes($ctrl.uiOptions.fromDate, 'from');
           $scope.$digest();
         }
 
         if (to) {
-          $ctrl.uiOptions.toDate = new Date(to);
+          $ctrl.uiOptions.toDate = getUTCDate(to);
           $ctrl.updateAbsencePeriodDatesTypes($ctrl.uiOptions.toDate, 'to');
           $scope.$digest();
         }
       }
+
+      function getUTCDate (date) {
+        var now = new Date(date);
```

Comment:
With moment we still get the old result of 2 failing test cases. I think we can keep current code.